style.css

```diff
@@ -0,0 +1,60 @@
+main {
+    background-color: rgba(204, 204, 204, 0.4);
+    padding-top: 20px;
+    font-family: "Segoe UI", "Roboto", "Helvetica Neue", Arial, sans-serif;
+}
+
+h1 {
+    text-align: center;
+    font-size: 40px;
+}
+
+h1+p {
+    text-align: center;
+    font-size: 15px;
+}
+
+.photoWithDescription {
+    width: 98%;
+    margin: 0 auto;
+    background-color: white;
```

Comment:
Common color style for the whole code (rgba for this project, as you have started to use this color style).